CODE_OF_CONDUCT.md

```diff
@@ -134,4 +134,7 @@ If you wish to use this code of conduct for your own project, consider
 explicitly mentioning your moderation policy or making a copy with your
 own moderation policy so as to avoid confusion.
 
+
+## some add text
+=======
```

Comment:
Was this accidentally added?

Reply:
Yes, there were erroneous lines added to this when I created my fork.